### DISCUSSION TITLE
Splitting large threads into multiple html/text files

### DISCUSSION THREAD

**Discussion by nyettaurus** [ReagentX/imessage-exporter#298] [Ideas]
As conversations reach the hundreds of thousands, the files become so large that programs are sluggish/crash
An option to separate files greater than say, 5 mb into multiple html files would be neat
thanks for this program, works incredibly flawlessly.

**Comment by ReagentX** [comment]
You can [specify date ranges](https://github.com/ReagentX/imessage-exporter/blob/develop/imessage-exporter/README.md#how-to-use) to make monthly or even shorter collections of exports.
